src/main/java/com/gdschongik/gdsc/global/exception/ErrorCode.java

```diff
@@ -59,7 +59,9 @@ public enum ErrorCode {
 
     // Membership
     PAYMENT_NOT_VERIFIED(HttpStatus.CONFLICT, "회비 납부가 완료되지 않았습니다."),
-    ;
+
+    // Recruitment
+    WRONG_DATE_ORDER(HttpStatus.BAD_REQUEST, "시간 설정이 잘못되었습니다. 종료일이 시작일보다 같거나 앞설 수 없습니다.");
```

Comment:
```suggestion
    WRONG_DATE_ORDER(HttpStatus.BAD_REQUEST, "종료일이 시작일과 같거나 앞설 수 없습니다.");
```

++ WRONG_DATE_ORDER보다 좋은 이름 뭐 없을까요..
마땅한게 떠오르진 않는데 `WRONG_DATE_ORDER`는 message와 맞지 않는 것 같아요..

Reply:
이거 지금부터 조금 생각해보겠습니다!